lib/tasks/grda_warehouse.rake

```diff
@@ -320,7 +320,7 @@ namespace :grda_warehouse do
       Rake::Task['driver:hmis_external_apis:export:ac_clients'].invoke
     end
 
-    HmisExternalApis::ConsumeExternalFormSubmissionsJob.new.perform if HmisEnforcement.hmis_enabled? && GrdaWarehouse::DataSource.hmis.exists? && RailsDrivers.loaded.include?(:hmis_external_apis)
+    HmisExternalApis::ConsumeExternalFormSubmissionsJob.new.perform(encryption_key: ENV['EXTERNAL_FORMS_SHARED_AES_KEY']) if HmisEnforcement.hmis_enabled? && GrdaWarehouse::DataSource.hmis.exists? && RailsDrivers.loaded.include?(:hmis_external_apis)
```

Comment:
Any reason this can't be stored in the remote credential for the s3 connection instead of the ENV?  We could just add another column to hold it.  I've found storing the config in the DB to be more maintainable long-term.

Reply:
sure thing!

Reply:
@eanders I created a new type of credential "SymmetricEncryptionKey" and moved the decrypt method into it. Does that seem okay?